src/plugins/guided_onboarding/common/constants.ts

```diff
@@ -6,7 +6,71 @@
  * Side Public License, v 1.
  */
 
+import { GuideConfig } from './types';
+
 export const PLUGIN_ID = 'guidedOnboarding';
 export const PLUGIN_NAME = 'guidedOnboarding';
 
 export const API_BASE_PATH = '/api/guided_onboarding';
+
+export const testGuideConfig: GuideConfig = {
```

Comment:
[nit] Should we create a separate file for this so it's clear it's only used for testing purposes?